core/src/test/java/org/bitcoinj/core/SegwitAddressTest.java

```diff
@@ -203,26 +204,26 @@ public void invalidAddresses() {
 
     @Test(expected = AddressFormatException.InvalidDataLength.class)
     public void fromBech32_version0_invalidLength() {
-        SegwitAddress.fromBech32((Network) null, "BC1QR508D6QEJXTDG4Y5R3ZARVARYV98GJ9P");
+        addressParser.parseAddressAnyNetwork( "BC1QR508D6QEJXTDG4Y5R3ZARVARYV98GJ9P");
```

Comment:
Superfluous whitespace.

(multiple occurrences)

Reply:
Fixed in 04a578a7670dcfb9b209477d935bb205114d491b